src/test/java/org/fogbowcloud/manager/core/AaControllerTest.java

```diff
@@ -48,112 +48,174 @@ public void setUp() {
+    	
+    	//exercise/verify
     	this.aaController.authorize(Mockito.any(), Mockito.any());
     }
     
+    //test case: Check if authenticate method throws Unauthenticated exception when the federation token is valid. 
     @Test (expected = UnauthorizedRequestException.class)
-    public void testAuthorizeWhenUnauthorizedRequestExceptionOnInstance() throws UnauthorizedRequestException {
+    public void testAuthorizeWhenUnauthorizedRequestExceptionOnInstanceType() throws UnauthorizedRequestException {
+    	//set up
     	Mockito.when(this.authorizationPluginMock.isAuthorized(
     			Mockito.any(), 
     			Mockito.any(), 
     			Mockito.any(InstanceType.class))).thenReturn(false);
+    	
+    	//exercise/verify
     	this.aaController.authorize(Mockito.any(), Mockito.any(), Mockito.any(InstanceType.class));
     }
     
+    //test case: Check if authenticate method throws Unauthenticated exception when the federation token is valid. 
     @Test (expected = UnauthorizedRequestException.class)
-    public void testAuthorizeWhenUnauthorizedRequestExceptionOnOrder() throws FogbowManagerException {
+    public void testAuthorizeWhenUnauthorizedRequestExceptionOnOrderType() throws FogbowManagerException {
+    	//set up
     	Mockito.when(this.authorizationPluginMock.isAuthorized(
     			Mockito.any(), 
     			Mockito.any(), 
     			Mockito.any(Order.class))).thenReturn(false);
+    	
+    	//exercise/verify
     	this.aaController.authorize(Mockito.any(), Mockito.any(), Mockito.any(Order.class));
     }
     
+    //test case: Check if authenticate method throws Unauthenticated exception when the federation token is valid. 
     @Test (expected = UnauthorizedRequestException.class)
     public void testAuthorizeWhenUnauthorizedRequestException() throws FogbowManagerException {
+    	//set up
     	Mockito.when(this.authorizationPluginMock.isAuthorized(
     			Mockito.any(), 
     			Mockito.any())).thenReturn(false);
+    	
+    	//exercise/verify
     	this.aaController.authorize(Mockito.any(), Mockito.any());
     }
     
+    //test case: Check if create token is returning a valid token properly.
     @Test
     public void testGetLocalToken() throws FogbowManagerException, UnexpectedException {
+    	//set up
     	Mockito.when(this.localUserCredentialsMapperPluginMock.getCredentials(Mockito.any())).thenReturn(new HashMap<String, String>());
     	String accessId = "accessId";
     	User user = new User("id", "name");
     	Date date = new Date();
     	Map <String, String> attributes = new HashMap<String, String>();
-    	Token token = new Token(accessId, user, date, attributes);
-    	Mockito.when(this.localIdentityPluginMock.createToken(Mockito.any())).thenReturn(token);
-    	Assert.assertEquals(token, this.aaController.getLocalToken(null));
+    	Token expectedToken = new Token(accessId, user, date, attributes);
+    	Mockito.when(this.localIdentityPluginMock.createToken(Mockito.any())).thenReturn(expectedToken);
+    	
```

Comment:
although the test passed when the user is null, it is misleading (the very first it is that you are testing the error conditions because of null references. I'd replace by a properly constructed value.